py/server/deephaven/parquet.py

```diff
@@ -43,6 +43,8 @@ def _build_parquet_instructions(
     is_refreshing: bool = False,
     for_read: bool = True,
     force_build: bool = False,
+    generate_metadata_files: Optional[bool] = None,
+    base_name: Optional[str] = None,
     special_instructions: Optional[s3.S3Instructions] = None,
```

Comment:
Add support for passing lists of column lists to index. Consult Jianfeng for signature help. Here, and on old write method as well.